core/zoe/docs/zoe.md

```diff
@@ -0,0 +1,263 @@
+# Zoe: Offer-Safety Enforcement
+
+Note: Zoe is currently at the pre-alpha stage. It has not yet been
+formally tested or hardened.
+
+This guide assumes some knowledge of the [ERTP fundamentals](./README.md).
+
+## What is Zoe? 
+
+__For users__: Zoe guarantees that as a user of a smart contract, you
+will either get what you wanted or get a refund back, even if the
+smart contract is buggy or backdoored. (In fact, the smart contract
+never has access to your digital assets.)
+
+__For developers__: Zoe provides a safety net so you can focus on what
+your smart contract does best, without worrying about your users
+losing their assets due to a bug in the code that you wrote. Writing a
+smart contract on Zoe is easy: all of the Zoe smart contracts are
+written in the familiar language of JavaScript. 
+
+## Sounds like magic. How does it actually work?
+
+To use Zoe, we have to put things in terms of "offers". An offer is a
+statement about what you want and what you're willing to offer. It
+turns out, many smart contracts (apart from gifts and one-way
+payments) involve an exchange of digital assets that can be put in
+terms of offers. 
+
+In this version of Zoe, our offers are simple (see [our roadmap](TODO) for
+more complex offer types and their expected release). We can say
+things like, "I'll give you [three
+wood for two bricks](https://en.wikipedia.org/wiki/Catan)." We can
```

Comment:
I don't know the game. But outside the game, "wood" is a plural noun. "three wood" sounds as non-sensical as "three water"

Reply:
Yeah, hmm. I guess it depends on whether we want to keep the Catan example or not. I'll see what I can do